assets/airports/egkk.json

```diff
@@ -27,1222 +28,6313 @@
+                ["N51d28m48", "W000d06m55"],
+                ["N51d28m47", "W000d06m57"],
+                ["N51d28m46", "W000d07m00"],
+                ["N51d28m46", "W000d07m02"],
+                ["N51d28m45", "W000d07m04"],
+                ["N51d28m44", "W000d07m07"],
+                ["N51d28m44", "W000d07m10"],
+                ["N51d28m43", "W000d07m12"],
+                ["N51d28m43", "W000d07m15"],
+                ["N51d28m42", "W000d07m18"],
+                ["N51d28m42", "W000d07m20"],
+                ["N51d28m42", "W000d07m23"],
+                ["N51d28m42", "W000d07m26"],
+                ["N51d28m42", "W000d07m29"],
+                ["N51d28m42", "W000d07m32"],
+                ["N51d28m42", "W000d07m34"],
+                ["N51d28m43", "W000d07m37"],
+                ["N51d28m43", "W000d07m40"],
+                ["N51d28m44", "W000d07m42"],
+                ["N51d28m44", "W000d07m45"],
+                ["N51d28m45", "W000d07m48"],
+                ["N51d28m46", "W000d07m50"],
+                ["N51d28m46", "W000d07m52"],
+                ["N51d28m47", "W000d07m55"],
+                ["N51d28m48", "W000d07m57"],
+                ["N51d28m49", "W000d07m59"],
+                ["N51d28m51", "W000d08m01"],
+                ["N51d28m52", "W000d08m03"],
+                ["N51d28m53", "W000d08m05"],
+                ["N51d28m54", "W000d08m07"],
+                ["N51d28m56", "W000d08m09"],
+                ["N51d28m57", "W000d08m10"],
+                ["N51d28m59", "W000d08m12"],
+                ["N51d29m00", "W000d08m13"],
+                ["N51d29m02", "W000d08m14"],
+                ["N51d29m03", "W000d08m15"],
+                ["N51d29m05", "W000d08m16"],
+                ["N51d29m06", "W000d08m17"],
+                ["N51d29m08", "W000d08m18"],
+                ["N51d29m10", "W000d08m18"],
+                ["N51d29m12", "W000d08m19"],
+                ["N51d29m13", "W000d08m19"],
+                ["N51d29m15", "W000d08m19"],
+                ["N51d29m17", "W000d08m19"],
+                ["N51d29m17", "W000d08m19"],
+                ["N51d29m38", "W000d11m31"],
+                ["N51d29m39", "W000d11m34"],
+                ["N51d29m39", "W000d11m37"],
+                ["N51d29m40", "W000d11m39"],
+                ["N51d29m40", "W000d11m42"],
+                ["N51d29m41", "W000d11m45"],
+                ["N51d29m42", "W000d11m47"],
+                ["N51d29m42", "W000d11m49"],
+                ["N51d29m43", "W000d11m52"],
+                ["N51d29m44", "W000d11m54"],
+                ["N51d29m45", "W000d11m56"],
+                ["N51d29m47", "W000d11m58"],
+                ["N51d29m48", "W000d12m00"],
+                ["N51d29m49", "W000d12m02"],
+                ["N51d29m50", "W000d12m04"],
+                ["N51d29m52", "W000d12m06"],
+                ["N51d29m53", "W000d12m07"],
+                ["N51d29m55", "W000d12m09"],
+                ["N51d29m56", "W000d12m10"],
+               
```

Comment:
Due to new requirements that weren't being followed before, all arrival spawnPatterns using the same procedures should be consolidated into one, so that there is no possible conflicts outside the airspace